adapter/inventory.go

```diff
@@ -15,32 +15,16 @@
 package adapter
 
 import (
-	"istio.io/mixer/adapter/denyChecker"
-	"istio.io/mixer/adapter/genericListChecker"
-	"istio.io/mixer/adapter/ipListChecker"
 	"istio.io/mixer/adapter/kubernetes"
 	"istio.io/mixer/adapter/memQuota"
```

Comment:
why leave legacy `memQuota` ?

Reply:
legacy memQuota is enforcing quota at present.
That switch will happen as a bug later.